### DISCUSSION TITLE
LXQT tag has far old/ wrong sub repositories linked

### DISCUSSION THREAD

**Discussion by thiyashoksanju** [lxqt/lxqt#2260] [General]
Hi,
I am currently trying to build LXQT v0.17.0. I followed the pattern of **Github -> lxqt -> lxqt -> releases -> 0.17.0**  path to identify the source code link
[Link](https://github.com/lxqt/lxqt/tree/0.17.0). And took individual repositories (liblxqt) from the above link, believing that every repositories belong to the lxqt 0.17.0 release.

After building and installing liblxqt, restarted my PC and my desktop environment is not loading (No code changes done here).

On the next time i followed same logic but instead of going with parent lxqt tag i went for the liblxqt repository **Github -> lxqt -> liblxqt -> releases -> 0.17.0** [Link](https://github.com/lxqt/liblxqt/tree/0.17.0). This time after build & install the issue not coming. So i compared above codes and found that both are different.

Analysed the change log files from both code and found that the liblxqt code downloaded from 0.17.0 tag of lxqt parent repository (first link) has older version(0.11.0) codes linked. 

I believe every repository attached below the 0.17.0 of lxqt parent must be the exact codes of that release (0.17.0 and so). 

It would be great if i get fix for this incorrect source code leading.

Here are few more details:
OS version : Lubuntu 21.10 (Impish Indri)
Kernel version : 5.13.0-25 generic
LXQT version : 0.17.0

**Comment by tsujan** [comment]
It was explained [before](https://github.com/lxqt/lxqt/discussions/2252#discussioncomment-2936911):

https://github.com/lxqt/lxqt is only for a one-step compilation of the latest git. One shouldn't use it for another purpose.
